README.adoc

```diff
@@ -473,6 +479,34 @@ doc:2:test_obvious_matching_with_assert_not_matches()
 	Running test_obvious_notmatching_with_assert_not_matches ... SUCCESS
 ```
 
+=== *assert_within_delta*
+
+    assert_within_delta <expected num> <actual num> <max delta> [message]
+
+Asserts that the expected num matches the actual num up to a given max delta.
+Given an expectation of 5 and a delta of 2 this would match 3, 4, 5, 6, and 7:
+
+```test
+test_matches_within_delta(){
+  assert_within_delta 5 3 2
+  assert_within_delta 5 4 2
+  assert_within_delta 5 5 2
+  assert_within_delta 5 6 2
+  assert_within_delta 5 7 2
+}
+test_does_not_match_within_delta(){
+  assert_within_delta 5 2 2
```

Comment:
Is there any value in adding a test for failing if outside of the delta in the other direction?

Reply:
There is definitely value :) I needed that tests for the implementation at some point.